include/nuttx/sched.h

```diff
@@ -506,6 +506,10 @@ struct task_group_s
 
   struct group_shm_s tg_shm;        /* Task shared memory logic             */
 #endif
+
+  /* Virtual memory mapping info ********************************************/
+
+  FAR struct mm_map_s tg_mm_map;    /* Task mmappings */
```

Comment:
remove FAR and include nuttx/mm/map.h in this file, and remove the same inclusion from group_create.c and group_leave.c